cms/locale/cy/LC_MESSAGES/django.po

```diff
@@ -434,105 +439,111 @@ msgstr ""
 
 #: cms/jinja2/templates/pages/statistical_article_page.html:31
 msgid "View previous releases"
-msgstr ""
+msgstr "Gweld datganiadau blaenorol"
 
 #: cms/jinja2/templates/pages/statistical_article_page.html:36
 msgid "Not the latest release"
 msgstr "Cyhoeddiad nesaf"
 
 #: cms/jinja2/templates/pages/statistical_article_page.html:37
 msgid "View latest release"
-msgstr ""
+msgstr "Gweld y datganiad diweddaraf"
 
 #: cms/jinja2/templates/pages/statistical_article_page.html:71
 msgid "To be announced"
-msgstr ""
+msgstr "I'w gyhoeddi"
 
-#: cms/jinja2/templates/pages/statistical_article_page.html:138
+#: cms/jinja2/templates/pages/statistical_article_page.html:137
 msgid "Show detail"
-msgstr ""
+msgstr "Dangos manylion"
 
-#: cms/jinja2/templates/pages/statistical_article_page.html:139
+#: cms/jinja2/templates/pages/statistical_article_page.html:138
 msgid "Hide detail"
 msgstr ""
 
-#: cms/jinja2/templates/pages/statistical_article_page.html:141
+#: cms/jinja2/templates/pages/statistical_article_page.html:140
 msgid "Corrections and notices"
-msgstr ""
+msgstr "Corrections and notices"
```

Comment:
This matches what we got in the PO file, but for other untranslated strings we leave `msgstr` as `""` so let's replace that here. I think it's good for consistency + potentially avoiding issues if we tried to write scripts or use translation tools.